docs/application/native/getting-started/mobile/first-app.md

```diff
@@ -28,119 +22,89 @@ Studio and installing the created application on the emulator or target device.
 
     This step shows how you can run the application on the emulator or a real target device.
 
-5. [Build a UI](#build_ui).
+5. [Design a UI](#build_ui).
 
-    This step shows how you can make small alterations to the
-    application UI to improve the usability of your application.
+    This step shows how you can create the application UI and make small alterations to it to improve the usability of your application.
 
-When you are developing a more complex application, you can take
-advantage of the [native tools included in the Tizen
-Studio](../../../tizen-studio/native-tools/cover-native-n.md)
-to ease the tasks of creating functionality and designing the
-application UI.
+When you are developing a more complex application, you can take advantage of the [native tools included in the Tizen Studio](../../../tizen-studio/native-tools/cover-native-n.md) to ease the tasks of creating functionality and designing the application UI.
```

Comment:
Don't we need to change `cover-native-n.md` into `cover-native.md`?

Reply:
Done.
However, I will update all files under getting-started directory next time. Please review this PR except the files under getting-started.